src/scenes/AddEdit/add-edit-scene.jsx

```diff
@@ -67,6 +67,8 @@ export default class AddEditEventScene extends React.Component {
 
         if (this.props.refreshLabelsIfNeeded)
             this.props.refreshLabelsIfNeeded();
+
+        this.props.setPageTitlePrefix(this.state.eventData.id ? 'Edit Event' : 'Add Event');
```

Comment:
This breaks when editing a single occurrence of a recurring event for the first time, because the event won't have an id - just an sid and a rec_id. (while testing this I also ran into #87 )